### PR TITLE
Add responses

### PR DESCRIPTION
**Pull request by lisamsmith100**
Not enough time; need 5 more mins

### CODE REVIEW

**Comment by lisamsmith100** [comment]
adding FoF
comfort:3
clarity:3

File: diagnostic.md [hunk 1]
```diff
@@ -3,53 +3,65 @@
...
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+<!--$("li").first().next();
+    $("#OL").children("li");
```
Comment (jordanallain) [inline diagnostic.md:21]: Close, be careful about capitalization, also `ol` is not an ID so we don't need the `#` before it. Here's one way to do it:

`$('ol > li')`

File: diagnostic.md [hunk 2]
```diff
@@ -3,53 +3,65 @@
...
+    -->
 
 ## Question 3
 
 Write the jQuery code to navigate from one of the list item (`li`) DOM nodes to
 the `form` DOM node. (Assume the list item has a class of "listerItem").
 
-<!-- your answer starts here -->
+<!-- $("li").first().parent().parent().next();-->
```
Comment (jordanallain) [inline diagnostic.md:29]: Very close!

`$('li').parent().parent().find('form')`

File: diagnostic.md [hunk 3]
```diff
@@ -3,53 +3,65 @@
...
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
-<!-- your answer starts here -->
+<!-- The DOM is a model of the expressed code for html/css/js family of files  -->
 
-<!-- your answer ends here -->
+<!-- It's as wrapper for html/-->
```
Comment (jordanallain) [inline diagnostic.md:56]: Its not quite a wrapper, I like your first definition better.

**Comment by jordanallain** [comment]
4/8

Good work Lisa, check out the answers and let me know if something is unclear.

🇯🇴 🇯🇴 🇯🇴 🇯🇴

**Pull request closed without merging**